operator/pkg/validate/validate_values.go

```diff
@@ -80,8 +80,8 @@ func validateValues(node interface{}, path util.Path) (errs util.Errors) {
 
 // GenValidateError generates error with helpful message when input fails values.yaml schema validation
 func GenValidateError(mvs version.MinorVersion, err error) error {
-	vs := fmt.Sprintf("releaese-%s.%d", mvs.MajorVersion, mvs.Minor)
+	vs := fmt.Sprintf("release-%s.%d", mvs.MajorVersion, mvs.Minor)
 	return fmt.Errorf("the input values.yaml fail validation: %v\n"+
 		"check against https://github.com/istio/istio/blob/%s/operator/pkg/apis/istio/v1alpha1/values_types.proto for schema\n"+
-		"or run the command with --force flag to ignore the error", err, vs)
+		"or run the command with --force flag to ignore the error: %v\n", vs, vs, err)
```

Comment:
I think originally the error goes to the first %v and the vs goes to the %s of url, which matches

Reply:
My bad, I completely missed that in my testing, the only change which is required is typo fix.